tools/config_updater.py

```diff
@@ -593,7 +689,12 @@ def main(unused_argv):
   token = GetToken()
   config_lib.CONFIG.AddContext("Commandline Context")
   config_lib.CONFIG.AddContext("ConfigUpdater Context")
-  startup.Init()
+
+  if flags.FLAGS.subparser_name == "initialize":
+    startup.ConfigInit()
+    Initialize(config_lib.CONFIG, token=token)
```

Comment:
It's more maintainable and signals a clear intent that the code path should end. At the moment it's not obvious if there is further processing that needs to happen, you have to look at all the if's and think about what how they will be evaluated.